grammars/level5-Additions.lark

```diff
@@ -1,13 +1,19 @@
 program: _empty_program | _non_empty_program
 _empty_program: (_EOL | _SPACE)*
-_non_empty_program: _EOL* (command | error_invalid) _SPACE* (_EOL+ command _SPACE*)* _EOL* //lines may end on spaces and might be separated by many newlines
+
+_non_empty_program_skipping_disabled: _EOL* (command | error_invalid) _SPACE* (_EOL+ command _SPACE*)* _EOL* //lines may end on spaces and might be separated by many newlines
+_non_empty_program_skipping_enabled: _EOL* (command) _SPACE* (_EOL+ command _SPACE*)* _EOL*
 
 //placing assign after print means print is will print 'is' and print is Felienne will print 'is Felienne'
-command:+= assign_button | ifpressed_else | error_ifpressed_missing_else | ifelse | ifs -= error_invalid > assign
-_if_less_command: print | ask | turtle | assign_button | assign_list | add | remove | sleep | error_print_no_quotes | assign
+command_skipping_disabled:+= assign_button | ifpressed_else | error_ifpressed_missing_else | ifelse | ifs -= error_invalid > assign
```

Comment:
I don't fully grasp this name `command_skipping_disabled` Does it means commands that cannot be skipped?

Reply:
I realize that this is pretty confusing. Essentially, we need to have two slightly different grammars, one when we do not skip faulty code (rules with ..._skipping_disabled) and one grammar when we do skip faulty code (rules with ....skipping_enabled). Other rules that do not have these prefixes will work for both.

I have not yet found a more elegant solution to this. I will search a bit more and report if I find something.

Reply:
I found a solution! [Remove skipping rules from grammar](https://github.com/hedyorg/hedy/pull/4320/commits/6c47efbbd43dda47cfe63955fd20e0d3d11c9f56) fixes this. The grammar is now the same as in main.